grails-app/views/order/list.gsp

```diff
@@ -220,6 +223,19 @@
 					.datepicker('setDate', null);
 				});
 			});
+
+			function fetchOrdersDerivedStatus() {
```

Comment:
Oh interesting. When I wrote the "rude awakening" comment I was thinking that one way to deal with this would be to pull the data in asynchronously. So I guess we won't be rudely awakened.